app/frontend/actions.js

```diff
@@ -103,7 +99,8 @@ module.exports = ({setState, getState}) => {
       const sendAmount = {fieldValue: ''}
       const sendAddress = {fieldValue: ''}
       const sendResponse = ''
-      const usingTrezor = cryptoProvider === 'trezor'
+      const usingHwWallet = wallet.isHwWallet()
+      const hwWalletName = wallet.getHwWalletName()
```

Comment:
shouldn't this be called only if it is hw wallet?

Reply:
I just wanted to maintain simplicity of code, avoid redundant ifs and uniformity of all crypto providers. For mnemonic crypto provider the function returns undefined. But you're right. It's a bit tricky.